src/App.css

```diff
@@ -115,3 +119,12 @@
   width: 48px !important;
   right: 10px !important;
```

Comment:
you should not use !important, this is a bad practice

Reply:
According to @asdt560, this is needed to do responsive navPanel works, but it will be reviewed afterward.